.github/workflows/publish-autoinstrumentation-utils.yaml

```diff
@@ -0,0 +1,65 @@
+name: "Publish Utils for Auto-Instrumentation"
+
+on:
+  push:
+    paths:
+      - 'autoinstrumentation/utils/**'
+      - '.github/workflows/publish-autoinstrumentation-utils.yaml'
+    branches:
+      - main
+  pull_request:
+    paths:
+      - 'autoinstrumentation/utils/**'
+      - '.github/workflows/publish-autoinstrumentation-utils.yaml'
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Read version
+        run: echo "VERSION=$(cat autoinstrumentation/utils/version.txt)" >> $GITHUB_ENV
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-utils
```

Comment:
I would prefer to be more explicit here e.g. `ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-cp`